Meter.c

```diff
@@ -324,8 +324,10 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
          data->values[i] = data->values[i + 1];
 
       double value = 0.0;
-      for (uint8_t i = 0; i < this->curItems; i++)
-         value += this->values[i];
+      for (uint8_t i = 0; i < this->curItems; i++) {
+         if (!isnan(this->values[i]))
+            value += this->values[i];
+      }
```

Comment:
Braces for all or none of these lines; in this case without would be fine.

Alternatively `value += !isnan(this->values[i]) ? this->values[i] : 0;` would also be acceptable.